frontend/src/features/crawl-workflows/workflow-editor.ts

```diff
@@ -656,7 +656,7 @@ export class WorkflowEditor extends BtrixElement {
         }}
       >
         <sl-icon slot="suffix" name="chevron-double-right"></sl-icon>
-        ${msg("Review & Save")}
+        ${msg(html`Review & Save`)}
```

Comment:
It looks like from line 464 we can just use backticks without `html`? Should we omit for consistency?
```suggestion
        ${msg(`Review & Save`)}
```